lms/djangoapps/grades/signals/handlers.py

```diff
@@ -133,6 +133,14 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
             kwargs.get('points_possible'),
         )
     )
+    log.info(
+        u'Task: {}. Begin async calculation of subsection grades for {} in {} / {}'.format(
+            result.id,
+            kwargs['user_id'],
+            kwargs['course_id'],
+            kwargs['usage_id'],
+        )
+    )
```

Comment:
Interesting.  I don't have a particularly strong inclination for one over the other.  However, I am always thinking in terms of the Splunk queries we'll want to make.  It is desirable to have a consistent prefix to make it easy to query for all Grades related logging.  So, for example, if they all had a "Grades:" in them that's one way of querying.